my-sms/spec/factories/institutes.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'faker'
+
+FactoryBot.define do
+  factory :institute do
+    sequence(:name) { |n| "Institute #{n}" }
+  end
+  factory :institute_demo, class: :institute do
```

Comment:
specify as a trait maybe, or just update the name logic for the main/parent factory

Reply:
Refactored it into a trait in 52bc9b0b2a26ce81550b50e67027df0bb279029b
I like having the more deterministic factory for testing.